libs/libc/debug/Kconfig

```
@@ -0,0 +1,22 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+# These are library functions that may be overriden by architecture-
+# specific implementations.  Not all architectures support implementations
+# for every library function.
+
+menu "Library Debugging"
+
+config EABI_UNWINDER
+	bool "EABI Stack Unwinder"
+	default "n"
+	---help---
+		This option enables stack unwinding support in the nuttx
+		using the information automatically generated by the
+		compiler. The resulting nuttx image is slightly bigger but
```

```suggestion
		compiler. The resulting image is slightly bigger but
```
nit: I think `nuttx` may be omitted here

---

Or at least written correctly: NuttX :-D

---

Done